tests/utils/node-versions.ts

```diff
@@ -14,7 +14,7 @@ export const nodeVersions = [
 		)
 			? [
 				latestMajor('20.10.0'),
-				latestMajor('18.18.2'),
+				latestMajor('18.19.0'),
```

Comment:
should we add '18.8.2' here too?

Reply:
All good. Technically, tsx just supports the _latest_ LTS but it's nice to have tests to make sure baseline functionality.